sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/AvoidNotShortCircuitOperatorsForBooleanCheck.java

```diff
@@ -115,27 +115,18 @@ public final void visitToken(final DetailAST detailAST)
     {
 
         DetailAST currentNode = detailAST;
-        while (currentNode != null
```

Comment:
If you pass any of operators then soon or later you get `EXPR` as parent and that's why validating against other tokens doesn't make sense. Also null value should not be possible because I imagine that `null` is not passed to `visitiToken()` method. Am I right?